api/src/main/java/uk/gov/dhsc/htbhf/claimant/exception/PostcodesClientException.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.dhsc.htbhf.claimant.exception;
+
+/**
+ * Exception used when calling postcodes.io.
+ */
+public class PostcodesClientException extends RuntimeException {
```

Comment:
Possibly `PostcodesIoClientException` unless you want to use it for other postcode clients (OS Places for example)?